src/us/kbase/workspace/test/database/UtilTest.java

```diff
@@ -0,0 +1,240 @@
+package us.kbase.workspace.test.database;
+
+import java.util.Map;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Arrays;
+import java.util.ArrayList;
+
+import org.junit.Test;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.fail;
+
+import us.kbase.common.test.TestCommon;
+import us.kbase.workspace.database.Util;
+
+public class UtilTest {
+
+    public static final String EXP_EXC = "expected exception";
+    public static final String UNEXP_EXC = "unexpected exception";
+    public static final String INCORRECT_NULL_EMPTY = "incorrect null or empty";
+    public static final String NULL_STRING = null;
+    public static final String OK_STRING = "some string of stingy stringy strings strung together";
+    public static final String STRING_WITH_SPACES = "\n\n\n   string \t  \n";
+    public static final String FUN_UNICODE_STRING = "❌❉ ⨍∪ℕ ᏬᏁᎥς๏๔є sтя⌽ηg ❉❓❗";
```

Comment:
Just to double check - the `.length()` and codepoint count are different for this string?

Reply:
reworked this test -- the added character is one that is two characters long but only one code point.

Reply:
:+1: